main.tf

```diff
@@ -174,6 +174,7 @@ module "ecs_alb_service_task" {
   circuit_breaker_deployment_enabled = var.circuit_breaker_deployment_enabled
   circuit_breaker_rollback_enabled   = var.circuit_breaker_rollback_enabled
   permissions_boundary               = var.permissions_boundary
+  runtime_platform                   = var.runtime_platform
```

Comment:
This would need a dynamic

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition#runtime_platform

since the example shows that it's a block, not an argument

```hcl
  runtime_platform {
    operating_system_family = "WINDOWS_SERVER_2019_CORE"
    cpu_architecture        = "X86_64"
  }
```

Reply:
also, it's unclear from the documentation if we're allowed to add multiple `runtime_platform` blocks

Reply:
Please test this change

Reply:
@nitrocode this is an input to the module https://github.com/cloudposse/terraform-aws-ecs-alb-service-task

In that module it is constructed as a dynamic

Reply:
Ah I see. I assumed this input was added to the raw resource, not the upstream module. Thank you for correcting me.